src/cgame/cg_rocket_draw.cpp

```diff
@@ -917,6 +917,31 @@ class SpeedGraphElement : public HudElement
 	Color::Color backColor;
 };
 
+class PositionElement : public TextHudElement
+{
+public:
+	PositionElement( const Rml::String& tag ) :
+			TextHudElement( tag, ELEMENT_GAME, true )
+	{
+	}
+
+	void DoOnRender() override
```

Comment:
DoOnRender should only be used for elements that send drawing commands directly to the renderer, bypassing Rmlui rendering. This should be DoUnUpdate